doc/reference/configuration/cfg_binary_logging_snapshots.rst

```diff
@@ -136,4 +146,67 @@
     | Type: number
     | Default: 14400 seconds
     | Environment variable: TT_WAL_CLEANUP_DELAY
-    | Dynamic: **yes**
\ No newline at end of file
+    | Dynamic: **yes**
+
+
+.. _cfg_binary_logging_snapshots-wal_ext:
+
+.. confval:: wal_ext
+
+    Since version :doc:`2.11.0 </release/2.11.0>`.
+
+    (Enterprise Edition only) Allows you to add auxiliary information to each :ref:`write-ahead log <internals-wal>` record.
```

Comment:
```suggestion
    (**Enterprise Edition only**) Allows you to add auxiliary information to each :ref:`write-ahead log <internals-wal>` record.
```
Or maybe add a note - like in the [flightrec](https://github.com/tarantool/doc/pull/4007/files#diff-d5de9cc345b464252e9b7e67b6394f265b8bd6b183b08d98100a8667e606afc3R3) section ? Same for `secure_erasing`